api/framework/admin-announcements/bundle.md

```diff
@@ -0,0 +1,54 @@
+# admin-announcements
+
+Admin bundle for making announcements.
+
+## Description
+
+The bundle allows admins to create announcements to be displayed in announcements tab. Admins can optionally also set the announcement to be displayed in a pop-up. 
+
+## Parameters
```

Comment:
Are these the values that admins can set for an announcements? I first thought that these are parameters for some Oskari request or XHR request. Maybe add some small sentence what these are/change the title to make it more apparent?

Reply:
Yes. I will change the title a bit and add a small description.